fineftp-server/src/filesystem.cpp

```diff
@@ -9,7 +9,7 @@
 
 #define NOMINMAX
 #define WIN32_LEAN_AND_MEAN
-#include <Windows.h>
+#include <windows.h>
```

Comment:
Is there a reason for this edit? At least on my machine the file is:
`C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\Windows.h`
So with a capital `W`.

Reply:
When cross-compiling on Linux for Windows, this file is spelled with lower case w. I believe the compiler on Windows is insensitive to the case of the filename, so the change should be working on Windows, too, although I haven't had the possibility to try that.